src/countdown-timer/countdown-timer.style.tsx

```diff
@@ -0,0 +1,97 @@
+import styled, { css } from "styled-components";
+import { Color } from "../color";
+import { TextStyleHelper } from "../text";
+import { MediaQuery } from "../media";
+
+// =============================================================================
+// STYLE TYPES
+// =============================================================================
+
+interface CountdownStyleProps {
+    $opacity: boolean;
```

Comment:
rename this to something like `$visible` instead as the meaning of opacity true/false is ambiguous